layers/poi/class.sql

```diff
@@ -46,24 +48,28 @@ RETURNS TEXT AS $$
         WHEN subclass IN ('books','library') THEN 'library'
         WHEN subclass IN ('university','college') THEN 'college'
         WHEN subclass IN ('hotel','motel','bed_and_breakfast','guest_house','hostel','chalet','alpine_hut','camp_site') THEN 'lodging'
-        WHEN subclass IN ('chocolate','confectionery') THEN 'ice_cream'
+        WHEN subclass IN ('chocolate','confectionery', 'ice_cream') THEN 'ice_cream'
         WHEN subclass IN ('post_box','post_office') THEN 'post'
-        WHEN subclass IN ('cafe') THEN 'cafe'
+        WHEN subclass IN ('cafe', 'coffee') THEN 'cafe'
         WHEN subclass IN ('school','kindergarten') THEN 'school'
-        WHEN subclass IN ('alcohol','beverages','wine') THEN 'alcohol_shop'
+        WHEN subclass IN ('alcohol','beverages','wine', 'nightclub') THEN 'alcohol_shop'
         WHEN subclass IN ('bar','nightclub') THEN 'bar'
-        WHEN subclass IN ('marina','dock') THEN 'harbor'
-        WHEN subclass IN ('car','car_repair','taxi') THEN 'car'
-        WHEN subclass IN ('hospital','nursing_home', 'doctors', 'clinic') THEN 'hospital'
+        WHEN subclass IN ('marina','dock', 'boat_rental', 'boat_sharing', 'ferry_terminal') THEN 'harbor'
+        WHEN subclass IN ('car','car_repair','taxi', 'fuel', 'parking', 'car_wash', 'car_rental', 'charging_station', 'parking_space') THEN 'car'
+        WHEN subclass IN ('hospital','nursing_home', 'doctors', 'clinic', 'dentist', 'pharmacy', 'veterinary') THEN 'hospital'
         WHEN subclass IN ('grave_yard','cemetery') THEN 'cemetery'
         WHEN subclass IN ('attraction','viewpoint') THEN 'attraction'
         WHEN subclass IN ('biergarten','pub') THEN 'beer'
         WHEN subclass IN ('music','musical_instrument') THEN 'music'
         WHEN subclass IN ('american_football','stadium','soccer','pitch') THEN 'stadium'
-        WHEN subclass IN ('accessories','antiques','art','artwork','gallery','arts_centre') THEN 'art_gallery'
+        WHEN subclass IN ('antiques','art','artwork','gallery','arts_centre', 'theatre') THEN 'art_gallery'
         WHEN subclass IN ('bag','clothes') THEN 'clothing_store'
         WHEN subclass IN ('swimming_area','swimming') THEN 'swimming'
         WHEN subclass IN ('castle','ruins') THEN 'castle'
+        WHEN subclass IN ('restaurant', 'drinking_water') THEN 'restaurant'
+        WHEN subclass IN ('dive_centre', 'sports', 'dojo') THEN 'sport'
```

Comment:
good idea 👍 
We should also add the leisure=sports_centre.
And remove the shop=sports.